blockstore/splitstore/splitstore.go

```diff
@@ -1699,6 +1768,7 @@ func bytesToUint64(buf []byte) uint64 {
 	return i
 }
 
+// So, I get the name... I think? But maybe flip it and say `isStateObject` (or `notStateObject`)?
```

Comment:
hrm... maybe; not married to the name, but not sure the alternatives are any better either!

Reply:
Well, I assume "unitary" refers to "unit", but these aren't really "units". _Really_, we're looking to recurs through all "state" objects, skipping anything that's not a part of the state.

Reply:
Yeah, it refers exactly to that; it's a unit, no state attached in the blockstore.
Really, I don't particularly care how it's called, as long as it conveys enough information :)

Reply:
Well, it's a unit from the perspective of the state-tree. But I had to think _real_ hard to figure out why we were calling it that.

(but this also really doesn't matter)